rabbit-escape-engine/src/rabbitescape/engine/solution/NothingPlaying.java

```diff
@@ -0,0 +1,17 @@
+package rabbitescape.engine.solution;
+
+import rabbitescape.engine.World.CompletionState;
+
+final class NothingPlaying extends SolutionInterpreter
+{
+    public NothingPlaying()
+    {
+        super( SolutionParser.parse("1") );
+    }
+    
+    @Override
+    public SolutionTimeStep next( CompletionState worldState )
+    {
+        return new SolutionTimeStep( ++commandIndex, new TimeStepAction[] {} );
+    }
+}
```

Comment:
Can SolutionInterpreter just take a flag to tell it to emit empty time steps when it's past the end?  I don't like deriving from concrete classes where it can be avoided.